src/Tickets/Seating/app/blockEditor/filters.js

```diff
@@ -120,3 +121,15 @@ addFilter(
 	'tec.tickets.seating',
 	filterTicketIsAsc
 );
+
+addAction(
+	'tec.tickets.blocks.ticketUpdated',
+	'tec.tickets.seating',
+	useUpdateStatesSeatTypeByTicketId
```

Comment:
See my note in the file defining this function to inline the `dispatch` call directly.